### ISSUE TITLE
Flux prompt changes fail

### ISSUE THREAD

**Issue by jpXerxes** [bmaltais/kohya_ss#2709]
My recollection with doing SDXL loras I could change the prompt in prompt.txt and it would affect the next sample. Tried it a couple of times with Flux loras, and the console shows the message lines below and it hangs.

E:\kohya_ss\venv\lib\site-packages\torch\utils\checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
